app-misc/ckb/ckb-0.4.2-r1.ebuild

```diff
@@ -28,16 +27,20 @@ RDEPEND="${DEPEND}"
 DOCS=( CHANGELOG.md README.md )
 S="${WORKDIR}/${PN}-next-${PV}"
 
+PATCHES=(
+	"${FILESDIR}/${P}-gcc10.patch"
+)
+
 src_configure() {
 	local mycmakeargs=(
 		-DDISABLE_UPDATER=yes
 	)
-	cmake-utils_src_configure
+	cmake_src_configure
 }
 
 src_install() {
 	newinitd "${FILESDIR}"/ckb.initd ckb-daemon
-	cmake-utils_src_install
+	cmake_src_install
 }
 
 pkg_postinst() {
```

Comment:
You could use xdg.eclass for cache updating and then drop the pkg_postinst and pkg_postrm. Just watch for the ordering of inheritance